pkg/api/schema/schema.proto

```diff
@@ -1639,7 +1642,7 @@ service ImmuService {
 
   rpc Delete(DeleteKeysRequest) returns (TxHeader) {
     option (google.api.http) = {
-      post: "/db/delete"
+      post: "/db/deletekey"
```

Comment:
This endpoint was clashing with the one generated for the `DeleteDatabase()` rpc. 
Because of the fact that grpc-gateway was not invoked (see above), this didn't result in any error.